cluster/charts/crossplane/templates/namespace-personas-clusterroles.yaml

```diff
@@ -0,0 +1,136 @@
+{{- if .Values.personas.deploy }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}:stack-manager:ns:default:admin
+  labels:
+    rbac.crossplane.io/aggregate-to-namespace-default-admin: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - cache.crossplane.io
+  - compute.crossplane.io
+  - database.crossplane.io
+  - kubernetes.crossplane.io
+  - core.crossplane.io
+  - storage.crossplane.io
+  - workload.crossplane.io
+  resources:
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - stacks.crossplane.io
+  resources:
+  - stackinstalls
+  - stacks
+  verbs:
+  - "*"
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - rolebindings
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - delete
+  - deletecollection
+  - patch
+  - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}:stack-manager:ns:default:edit
+  labels:
+    rbac.crossplane.io/aggregate-to-namespace-admin: "true"
+    rbac.crossplane.io/aggregate-to-namespace-default-edit: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - cache.crossplane.io
+  - compute.crossplane.io
+  - database.crossplane.io
+  - kubernetes.crossplane.io
+  - core.crossplane.io
+  - storage.crossplane.io
+  - workload.crossplane.io
+  resources:
+  - "*"
+  verbs:
```

Comment:
same for these verbs, should they match the edit roleVerbs in https://github.com/displague/crossplane/blob/gh1049-stack-controllers-use-clusterrole/pkg/controller/stacks/stack/stack.go#L61?

Reply:
https://github.com/crossplaneio/crossplane/pull/1059#discussion_r360626059